api/net/Areas/Admin/Controllers/ReportController.cs

```diff
@@ -258,5 +270,17 @@ public async Task<IActionResult> Preview(ReportModel model)
 
         return new JsonResult(new ReportPreviewModel(subject, body, results));
     }
+
+    private string? GetImageContent(string? path)
+    {
+        path = string.IsNullOrWhiteSpace(path) ? "" : HttpUtility.UrlDecode(path).MakeRelativePath();
+        var safePath = Path.Combine(_storageOptions.GetUploadPath(), path);
+        if (!safePath.FileExists()) return null;
+
+        using FileStream fileStream = new(safePath, FileMode.Open, FileAccess.Read);
+        var imageBytes = new byte[fileStream.Length];
+        fileStream.Read(imageBytes, 0, (int)fileStream.Length);
+        return Convert.ToBase64String(imageBytes);
```

Comment:
I don't understand how this works?  You're converting the image into raw byte data and including it in the email?

Reply:
In this case we are getting the image content and display the image in the email.